service/src/lib.rs

```diff
@@ -389,7 +389,7 @@ pub trait Factory: Send + Sync {
 #[async_trait]
 pub trait ResourceBuilder<T> {
     fn new() -> Self;
-    async fn build(self, factory: &mut dyn Factory, runtime: &Runtime) -> Result<T, crate::Error>;
```

Comment:
I wonder if anything else in `shuttle_service` still needs tokio now :thinking:

Reply:
I've started on removing the runtime from resources, I'll can check in that one :+1: